.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "dart.runPubGetOnPubspecChanges": "always"
+  "dart.runPubGetOnPubspecChanges": "always",
+  "dart.lineLength": 80,
```

Comment:
This should not change in this PR, I'm not sure why this file is even checked in.

Reply:
I'm completely fine to get this one removed. The reason I added this line is that I use different line length for other dart/flutter projects of mine and it is convenient when the expected formatting settings are configured project-wide to avoid accidental missformatting by the IDE.